homeassistant/components/switcher_kis/cover.py

```diff
@@ -44,6 +44,11 @@ def async_add_cover(coordinator: SwitcherDataUpdateCoordinator) -> None:
         """Add cover from Switcher device."""
         if coordinator.data.device_type.category == DeviceCategory.SHUTTER:
             async_add_entities([SwitcherCoverEntity(coordinator)])
+        if (
+            coordinator.data.device_type.category
+            == DeviceCategory.SINGLE_SHUTTER_DUAL_LIGHT
+        ):
+            async_add_entities([SwitcherCoverEntity(coordinator, COVER1_ID)])
```

Comment:
```suggestion
            async_add_entities([SwitcherCoverEntity(coordinator, 0)])
```
Instead of using a string, pass here the cover index which is more reflecting the physical device, than you can later use this directly in the API calls. Future devices will pass the correct index also (1, 2)